TLM/TLM/Constants.cs

```diff
@@ -36,5 +38,7 @@ public static class Constants {
         public static IManagerFactory ManagerFactory => Manager.Impl.ManagerFactory.Instance;
 
         public static INotifier Notifier => TrafficManager.Notifier.Instance;
+
+        public static IRoadSignTheme ActiveTheme => RoadSignThemeManager.ActiveTheme;
```

Comment:
should I call it `ITextureTheme` and move it to `IUIFactory`.
That way I will need to introduce `UIFactory` with only one member but in future maybe we add stuff to it. like scale, transparency, ...
@Elesbaan70 what do you think?

Reply:
It makes sense to me, but I think @krzychu124 should definitely weigh in on that one, since he has the best sense of future product direction and what the API needs might be in this area.

Reply:
I'd say minimal viable is better than overengineered and "future-proof". You don't know what future brings, why design for it? Its good to add the possibility for future change in the design,  but don't design for requirements which do not exist yet.

Here's current use case: Some API user (1 addon?) wants some theme textures. 
Are there other use cases for the TM:PE theme?

Reply:
nope just my NC for now. I just want to fetch the textures from the active theme and that is all.